Scrapping/Scrapping.java

```diff
@@ -0,0 +1,180 @@
+   private String[] scrappingword(String URL) throws IOException {
+      // Connecting to the web page
+
+      Connection conn = Jsoup.connect(URL);
+      // executing the get request
+      Document doc = conn.get();
+      // Retrieving the contents (body) of the web page
+      String result = doc.body().text();
+
+      result = result.replaceAll("[^a-zA-Z' ']", "");
+      String[] WordsInUrl = result.split(" ");
+      return WordsInUrl;
+   }
+
+   ArrayList<LinkedHashMap<String, LinkedHashMap<String, Integer>>> CheckWordsCount(String UrlFile)
+         throws FileNotFoundException {
+      File file = new File(UrlFile);
+      try (Scanner inputUrl = new Scanner(file)) {
+         ArrayList<String> urlList = new ArrayList<>();
+
+         while (inputUrl.hasNextLine()) {
+            urlList.add(inputUrl.next());
+         }
+         if (urlList.isEmpty()) {
+            System.out.println("url.txt file is empty");
+            return null;
+         }
+         if (Words.isEmpty()) {
+            System.out.println("words.txt file is empty");
+            return null;
+         }
+         for (int index = 0; index < urlList.size(); index++) {
+            String URL = urlList.get(index);
+            HashMap<String, Integer> hm = new HashMap<String, Integer>();
+            // insertion of words in HashMap which you want to Check
+            for (String word : Words) {
+               hm.put(word, 0);
+            }
+
+            String[] WordsInUrl = scrappingword(URL);
+            for (String word : WordsInUrl) {
+               if (word.equals("")) {
+                  continue;
+               }
+               if (hm.containsKey(word)) {
+                  hm.put(word, hm.get(word) + 1);
+               }
+            }
+            // Extract the top three words with count in Descending Order
+            PriorityQueue<Map.Entry<String, Integer>> sortedSet = sortOrderByCount(hm);
+            int count = 0;
+            LinkedHashMap<String, Integer> wordsCount = new LinkedHashMap<String, Integer>();
+            for (Map.Entry<String, Integer> map : sortedSet) {
+               if (count < 3) {
+
+                  wordsCount.put(map.getKey(), map.getValue());
+                  count++;
+               }
+               if (allWordWithCount.containsKey(map.getKey())) {
+                  allWordWithCount.put(map.getKey(), allWordWithCount.get(map.getKey()) + map.getValue());
+               } else {
+                  allWordWithCount.put(map.getKey(), map.getValue());
+               }
+
+            }
+            LinkedHashMap<String, LinkedHashMap<String, Integer>> h = new LinkedHashMap<>();
+            h.put(URL, wordsCount);
+            AllUrlWordsWithCount.add(h);
+         }
+      } catch (IOException exception) {
+         System.out.println(exception);
+      }
+
+      return AllUrlWordsWithCount;
+
+   }
+
+   public static void main(String args[]) throws FileNotFoundException {
```

Comment:
Don't just throw exceptions in the main method. All exceptions should be handled with try and catch block. And try to find maximum exceptions and errors that can occur in your code and handle them. Do this throughout the modules.